chapters/classes.tex

```diff
@@ -1397,6 +1397,12 @@ \section{Predefined Types and Classes}\label{predefined-types-and-classes}
 The definitions use \lstinline!RealType!\indexinline{RealType}, \lstinline!IntegerType!\indexinline{IntegerType}, \lstinline!BooleanType!\indexinline{BooleanType}, \lstinline!StringType!\indexinline{StringType}, \lstinline!EnumType!\indexinline{EnumType} as mnemonics corresponding to machine representations.
 These are called the \firstuse[primitive type]{primitive types}.
 
+\begin{definition}[Fall-back value]\label{def:fall-back-value}\index{fall-back value}
+When the start-value is needed and the \lstinline!start!-attribute is not provided, the fall-back value is used instead.
+The fall-back value can be be used as guess-value in system of equations and when initializing variables in algorithm sections.
```

Comment:
It becomes wrong to say that the start-value is needed if it is perfectly fine to use the fallback value instead.
```suggestion
In situations where the \lstinline!start!-attribute would apply if provided, but the attribute is not provided, the \emph{fallback} value may be used instead.
Use is restricted to guess-values in systems of equations and initialization of variables in algorithm sections.
```

So we don't allow it for adding equations to the initialization problem?

Reply:
Hmm... Actually we should allow it for the initialization problem as well. In particular for non-clocked when-statements it makes a lot of sense to forget to give them a start-value and using the fallback value (especially in the common case where it is 0). Reformulating - in particular to ensure that the default case is handled as previously where start=0 was present.